databases/week2/homework.sql

```diff
@@ -0,0 +1,46 @@
+-- Part 2: School database
+
+-- Create a new database containing the following tables:
+
+-- Class: with the columns: id, name, begins (date), ends (date)
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+
+CREATE DATABASE school;
+
+USE school;
+
+CREATE TABLE
+    Class(
+        Id INT,
+        Name VARCHAR(255),
+        Begins DATE,
+        Ends DATE
+    );
+
+CREATE TABLE
+    Student(
+        Id INT NOT NULL PRIMARY KEY AUTO_INCREMENT,
+        Name VARCHAR(255),
+        Email VARCHAR(255),
+        Phone INT,
+        Class_id INT,
+        FOREIGN KEY (Class_id) REFERENCES Class(Id)
+    );
+
+ALTER TABLE
+    Class Modify column Id INT NOT NULL PRIMARY KEY AUTO_INCREMENT;
```

Comment:
MODIFY COLUMN should be all in capitals😊

Reply:
I changed it 😀
Thank you !